### ISSUE TITLE
Appium installs WinAppDriver in the wrong location

### ISSUE THREAD

**Issue by lovettchris** [microsoft/WinAppDriver#1966]
I kept getting "message": "An unknown error occurred in the remote end while processing the command." until I fully uninstalled the WinAppDriver then installed it again from your github releases.  I noticed your installer places it in `C:\Program Files (x86)\Windows Application Driver` which works, but appium was placing it in `C:\Program Files\Windows Application Driver`.

**Comment by RakeshDangi-slt** [comment]
Hi @lovettchris,
It is intended to be installed on that location only, and for the error have you enabled the developer mode?
also if you are using it in conjunction with Appium driver you need to install its driver from the Appium command line "Appium driver install......"
We have an already developed solution for the same, you can check it out over here "https://soliterata.com/".

**Comment by lovettchris** [comment]
Yep, I already checked developer mode is enabled, but `Appium driver install....` is how I installed it the first time, but nothing would work.  I kept getting that error message until I manually uninstalled WinAppDriver, and installed your .msi which put it in `C:\Program Files (x86)` then appium magically started working properly.

**Comment by RakeshDangi-slt** [comment]
yes, you need to have the win app driver installed, you are doing it correctly.

**Comment by Diarmaid89** [comment]
Hi, I'm encountering this same issue. Appium installed it to 'C:\Program Files\Windows Application Driver' originally. Now I'm trying to run this on a different machine but keeping the program paths the same. Running 'WindowsApplicationDriver_1.2.1.msi' force installs it to the 'C:\Program Files (x86)\Windows Application Driver' location.

To be clear, the installer offers the option to change to install path or create a new folder somewhere outside the Program Files (x86) folder, but infuritaingly, it ignores whatever you choose and installs to the 'C:\Program Files (x86)\Windows Application Driver' anyway.
